test/assets/unrunnableTestCases.csv

```diff
@@ -1626,7 +1618,7 @@ K-NumericAdd-35,AssertionError: Expected executing the XPath "3 + (4, error())"
 K-NumericAdd-36,AssertionError: expected [Function] to throw error matching /FOER0000/ but got 'XPST0017: Function error with arity of 0 not registered. Did you mean "Q{http://www.w3.org/2001/XMLSchema}error (xs:anyAtomicType?)"?'
 K-NumericAdd-41,AssertionError: Expected executing the XPath "error() + 3" to resolve to one of the expected results, but got AssertionError: expected [Function] to throw error matching /FOER0000/ but got 'XPST0017: Function error with arity of 0 not registered. Did you mean "Q{http://www.w3.org/2001/XMLSchema}error (xs:anyAtomicType?)"?', AssertionError: expected [Function] to throw error matching /XPST0005/ but got 'XPST0017: Function error with arity of 0 not registered. Did you mean "Q{http://www.w3.org/2001/XMLSchema}error (xs:anyAtomicType?)"?'.
 K-NumericAdd-42,AssertionError: expected [Function] to throw error matching /FOER0000/ but got 'XPST0017: Function error with arity of 0 not registered. Did you mean "Q{http://www.w3.org/2001/XMLSchema}error (xs:anyAtomicType?)"?'
-op-numeric-add-1,Error: Not implemented: Intermediate clauses in flwor expressions are not implemented yet
+op-numeric-add-1,AssertionError:           for $x in (1, xs:decimal(2), xs:float(3), xs:double(4), xs:untypedAtomic(5))           for $y in (1, xs:decimal(2), xs:float(3), xs:double(4), xs:untypedAtomic(5))          return ($x + $y)       : expected false to be true
```

Comment:
I checked it using the playground and the problem was that the assertion was wrong. [This](https://github.com/w3c/qt3tests/blob/23f29d53b0e71379d7008bb06397f1478f8579a4/op/numeric-add.xml#L1344) is the test. fontoxpath evaluates this to double instead of float.